cirq-core/cirq/qis/measures_test.py

```diff
@@ -207,7 +206,10 @@ def test_von_neumann_entropy():
     probs = np.random.exponential(size=N)
     probs /= np.sum(probs)
     mat = U @ (probs * U).T.conj()
-    np.testing.assert_allclose(cirq.von_neumann_entropy(mat), scipy.stats.entropy(probs, base=2))
+
+    np.testing.assert_allclose(
+        cirq.von_neumann_entropy(mat), -np.sum(probs * np.log(probs) / np.log(2))
```

Comment:
nit: can use np.log2
```suggestion
        cirq.von_neumann_entropy(mat), -np.sum(probs * np.log2(probs))
```